python/ray/dashboard/modules/metrics/dashboards/train_dashboard_panels.py

```diff
@@ -5,55 +5,63 @@
     Target,
 )
 
+CHECKPOINT_REPORT_TIME_PANEL = Panel(
+    id=1,
+    title="Checkpoint Report Time",
+    description="Time taken to report a checkpoint to storage.",
+    unit="seconds",
+    targets=[
+        Target(
+            expr="sum(ray_train_report_total_blocked_time_s{{ray_train_worker_world_rank=~'$TrainWorkerWorldRank', {global_filters}}}) by (ray_train_run_name, ray_train_worker_world_rank)",
```

Comment:
if we add a worker id filter as well, we can scope the graph to a single worker of a single attempt since worker id is randomly generated.

Reply:
I understand now

<div><img src="https://media0.giphy.com/media/eJppgugSxYVNb4J5F3/giphy.gif?cid=5a38a5a28s0vr7xh04b08ejn00ac0paj138skybhw0dvuvna&amp;ep=v1_gifs_search&amp;rid=giphy.gif&amp;ct=g" style="border:0;height:225px;width:300px"/><br/></div>